CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## Master
+
+#### Breaking
+
+* None.
+
+#### Enhancements
+
+* Adds `discouraged_optional_collection` opt-in rule to encourages the return
```

Comment:
s/encourages the return/encourage the use/

Reply:
Thanks, @jpsim! 

I'll have to refactor/rewrite this rule. There are some cases that are really hard to cover using regular expression. For instance:

```swift
fun foo(param: [String: ↓[String: String]?])
```

vs.

```swift
fun foo(param: ↓[String: [String: String]]?)
```

But thanks, I'll update the changelog entry.